jvb/src/main/resources/reference.conf

```diff
@@ -192,6 +192,32 @@ videobridge {
       #}
     ]
   }
+  # Configuration related to pushing statistics to callstats.io.
+  callstats {
```

Comment:
Seems like this should be scoped under `stats`?  (Though not under `transports`, as I know this isn't the transport config but the general CallStats config)

Reply:
I thought about this, but wasn't sure. I don't mind changing it, will take a look in the morning.